library-kotlin-android-extensions/src/main/java/com/xwray/groupie/kotlinandroidextensions/Item.kt

```diff
@@ -0,0 +1,14 @@
+package com.xwray.groupie.kotlinandroidextensions
+
+import android.view.View
+import com.xwray.groupie.Item
+
+abstract class Item : Item<ViewHolder> {
+
+    constructor() : super()
+    constructor(id: Long) : super(id)
```

Comment:
nitpick, can't you do default param for id similar to something like 
```class MyView  constructor(context: Context, attrs: AttributeSet? = null, defStyle: Int = 0)

Reply:
It's a good question, but the default implementation to auto-generate the id is in the Java base class's constructor and the counter is currently private.  

    public Item() {
        this(ID_COUNTER.decrementAndGet());
    }

I guess you could refactor to expose the counter as protected or something but it feels bad to me.  I really didn't want anyone else's subclasses fucking with the counter. Thoughts?

Reply:
I'm still a bit noobish on kotlin but I think `internal` visibility would mean it is visible within your module but not to consumers of the library

Reply:
No, the base class is in Java ... but package private might do!

Reply:
OK, it will, but then I have to republish the base lib so you're right, but I'm lazy

Reply:
Ignore me 😀 just causing trouble for no reason  Next bad suggestion: kotlin 1.2.0-beta-88?

Reply:
i think it's a toss up between 1.6-eap and 1.2-beta :) . at the rate they're going it's possible 1.2 will go final in time for groupie 2.0!